dpsim/src/Utils.cpp

```diff
@@ -54,7 +54,7 @@ CommandLineArgs::CommandLineArgs(int argc, char *argv[],
 		{ "start-in",		required_argument,	0, 'i', "SECS", "" },
 		{ "solver-domain",	required_argument,	0, 'D', "(SP|DP|EMT)", "Domain of solver" },
 		{ "solver-type",	required_argument,	0, 'T', "(NRP|MNA)", "Type of solver" },
-		{ "solver-mna-impl", required_argument, 0, 'U', "(EigenDense|EigenSparse|CUDADense|CUDASparse)", "Type of MNA Solver implementation"},
+		{ "linear-solver-impl", required_argument, 0, 'U', "(DenseLU|SparseLU|KLU|CUDADense|CUDASparse)", "Type of MNA Solver implementation"},
```

Comment:
Still should be "Type of direct linear solver implementation"

Reply:
done